src/main/java/eu/darkbot/utils/EquippableItem.java

```diff
@@ -0,0 +1,6 @@
+package eu.darkbot.utils;
+
+public interface EquippableItem {
+
+    String getExceptionMessage();
```

Comment:
Definetly not. An item does not have an "exception message", if anything give it a name or something more generic